File: RecipeJetpack/app/src/main/java/com/example/recipejetpack/screen/recipe/paging/model/Result.kt
package com.example.recipejetpack.screen.recipe.paging.model

import com.google.gson.annotations.SerializedName

data class Result(
    var pk: Long?,
    var title: String?,
    var publisher: String?,
    @SerializedName("featured_image")
    var featuredImage: String?,
    var rating: Int?,
    @SerializedName("source_url")
    var sourceUrl: String?,
    var description: String?,
    @SerializedName("cooking_instructions")
    var cookingInstructions: Any?,
    var ingredients: List<String>?,
    @SerializedName("date_added")
    var dateAdded: String,
    @SerializedName("date_updated")
    var dateUpdated: String,
    @SerializedName("long_date_added")
    var longDateAdded: Long,
    @SerializedName("long_date_updated")
    var longDateUpdated: Long
)

File: demo_flutter/lib/screen/recipe/model/result.dart
import 'package:freezed_annotation/freezed_annotation.dart';

part 'result.freezed.dart';
part 'result.g.dart';

@freezed
class Result with _$Result {
  const factory Result({
    double? pk,
    String? title,
    String? publisher,
    String? featured_image,
    int? rating,
    String? source_url,
    String? description,
    dynamic cooking_instructions,
    List<String>? ingredients,
    String? date_added,
    String? date_updated,
    double? long_date_added,
    double? long_date_updated
  }) = _Result;

  factory Result.fromJson(Map<String, dynamic> json) => _$ResultFromJson(json);
}